emitter/emitter.go

```diff
@@ -687,3 +694,22 @@ func emitMovementStatement(movementStmt *ast.MovementStatement) string {
 	sb.WriteString(fmt.Sprintf("\t%s\n", terminator))
 	return sb.String()
 }
+
+func emitMartStatement(martStmt *ast.MartStatement) string {
+	terminator := "ITEM_NONE"
+	var sb strings.Builder
+	sb.WriteString("\t.align 2\n")
```

Comment:
I think `.align 1`, rather than `.align 2`, is all you need, since it has to align to the nearest 2-byte boundary.  I'm like 80% confident in that.

Reply:
I asked in pret earlier if this was required and @GriffinRichards says not having it can cause issues with items with ids taking up both bytes. I do not know the details but you can ask him.

Reply:
The alignment itself is, but as Shanty says it doesn't specifically need to be `.align 2`. If you're referring to <https://github.com/pret/pokeemerald/pull/1490>, I used `.align 2` rather than `.align 1` because all pre-existing directives for pokemarts use the former